R/cookies.R

```diff
@@ -342,7 +342,7 @@ Shiny.addCustomMessageHandler('analytics-consent', function(msg){
 #' "ABCDE12345"
 #'
 #' @family cookies
-#' @return a html div, containing standard cookises content for a public R
+#' @return a html div, containing standard cookies content for a public R
```

Comment:
I think HTML should be in caps - it's not in the style guide but this is consistent with how it's used elsewhere

Reply:
Fixed this, and a few other instances in my last commit